pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -108,7 +108,7 @@ func preempt(ctx context.Context, fh framework.FrameworkHandle, state *framework
 		return "", err
 	}
 
-	if !podEligibleToPreemptOthers(pod, fh.SnapshotSharedLister().NodeInfos()) {
+	if !podEligibleToPreemptOthers(pod, fh.SnapshotSharedLister().NodeInfos(), m) {
```

Comment:
You can just pass a *status: `m[pod.Status.NominatedNodeName]`, which will be evaluated to nil if either pod.Status.NominatedNodeName == "" or m[nnn] does not exist.

Reply:
And please add some comments to explain the case.

Reply:
got it, will do